zerver/webhooks/uptimerobot/doc.md

```diff
@@ -0,0 +1,41 @@
+Receive Zulip notifications from Uptime Robot!
+
+1. {!create-stream.md!}
+
+1. {!create-a-bot-indented.md!}
+
+    Construct the URL for the {{ integration_display_name }}
+    bot using the bot's API key and the desired stream name:
+
+    `{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream=stream%20name&`
+
+    Modify the parameters of the URL above, where `api_key` is the API key
+    of your Zulip bot, and `stream` is the URL-encoded stream name you want the
+    notifications sent to. If you do not specify a `stream`, the bot will
+    send notifications via PMs to the creator of the bot.
```

Comment:
I think this text is already in the macros above, right?

Reply:
@timabbott No, it isn't it, we usually use `{!create-bot-construct-url-indented.md!}` but a specific requirement for uptime robot webhook implementation is to have an `&` at the end of the URL as that is how it passes the payload. So I had to manually add that text after just using `{!create-a-bot-indented.md!}` to just show how to create the bot in zulip. I can mention this in the commit message if you want so we know when looking back at it.

Reply:
Yeah, that detail belongs in the commit message.